### ISSUE TITLE
Downloading other formats doesn't work if you've already downloaded a single file html

### ISSUE THREAD

**Issue by roedoejet** [ReadAlongs/Studio-Web#112]
There must be something that gets added to the RAS (either image or translations) that the RAS API doesn't like because you can download all file formats if you haven't downloaded a single file html yet. As soon as you download the single file html (which adds translation and image elements to the ras), then the other formats don't work.

**Comment by dhdaines** [comment]
Hmm. Curiously, if you then download the single file HTML a second time, downloading other formats will work.  It definitely has something to do with the image/translation insertions because the HTML files are different (and the difference is in the RAS XML)

**Comment by dhdaines** [comment]
And the culprit is... **NAMESPACES** (NEWMAN!!!).  This is in the one that doesn't get accepted:

```xml
<s xmlns="http://www.w3.org/1999/xhtml" do-not-align="true" id="t0b0d0p0s0" class="sentence__translation editable__translation" xml:lang="eng"></s>
```
whereas the acceptable one has:

```xml
<s do-not-align="true" xml:lang="eng" id="t0b0d0p0s0tr" class="sentence__translation">Hello.</s>
```

**Comment by dhdaines** [comment]
fixed in #114 (I hope)
